CONTRIBUTING.md

```diff
@@ -140,7 +140,7 @@ This way when someone hovers over the image, the description of the image appear
 
 All screenshots should be taken using Atom's default themes and settings.
 
-[pngcrush]: http://pmt.sourceforge.net/pngcrush/
+[pngcrush]: http://pmt.sourceforge.net/pngcrush
```

Comment:
I don't understand the reason behind this change — if you navigate to the proposed URL, you end up at the original URL (with the appended forward slash). The host actually issues a 301 redirect if you attempt to navigate to the proposed URL.

Please justify this change since we're now linking to an endpoint that has a 301 permanent redirect.